### ISSUE TITLE
My Solar App not working

### ISSUE THREAD

**Issue by luigimoran** [richardlevy/solis-cloud-scraper#10]
Hi,

Can You please help me setting up the "My Solar" apple watch app? I'm trying the solis cloud scraper method. I can already run the scraper.js and can even already query in the cmd using the curl method. But I can't setup properly it to the Iphone app. I'm trying to input the http://[ip address]:port/data on the solis scraper uri part, username and solis scraper password, but when I test it, it saying Cannot Connect. What should I do? What is it that I'm doing wrong?

Best Regards and Thank You,

Luigi

**Comment by richardlevy** [comment]
Hi

for the Solis Scraper URI, just have http://[ip address]:port

Thanks
Rich

**Comment by luigimoran** [comment]
Hi Rich,

I've tried inputting the http://[ip address of my computer]:port to the Solis Scraper URI. When I tried testing it, the software crashed. When I try to enable it, the iPhone app can't be opened anymore. I'm running the node scraper.js command on my computer. I've tried the curl both from the computer that is running and to other computer and it is working. It's just not working to my Iphone SE 2020.

I've also commented out the part of the "Battery charge level now", "Battery charging today" and "Battery discharge today" under the scrapeData function as it is saying an error 
Error at stage Getting battery charge
Error - Cannot read properties of null (reading 'getProperty')
Error fetching data - using previous data

Thanks,
Luigi
